CommentUpdater/src/main/java/org/eupathdb/sitesearch/data/comments/CommentUpdaterCli.java

```diff
@@ -32,33 +33,35 @@
     String getCommentSchema() { return getThird(); }
   }
 
-  public static void main(String[] args) throws Exception {
+  void init(CommentSolrDocumentFields docFields, CommentUpdaterSql updaterSql) throws Exception {
     Config config = parseEnv();
     try (DatabaseInstance commentDb = new DatabaseInstance(config.getDbConfig())) {
-      CommentUpdater updater = new CommentUpdater(config.getSolrUrl(), commentDb, config.getCommentSchema());
+      CommentUpdater updater = 
+          new CommentUpdater(config.getSolrUrl(), commentDb, config.getCommentSchema(),
+              docFields, updaterSql );
       updater.syncAll();
     }
   }
 
-  private static Config parseEnv() {
+  private Config parseEnv() {
     final var env = System.getenv();
 
-    if (!(env.containsKey(ENV_DB_CONNECT) && env.containsKey(ENV_DB_PASS)
-        && env.containsKey(ENV_DB_SCHEMA) && env.containsKey(ENV_DB_USER)
-        && env.containsKey(ENV_SOLR_URL))
+    if (!(env.containsKey(getEnvDbConnect()) && env.containsKey(getEnvDbPass())
```

Comment:
How is this related to the "requiredFields" method above?  Maybe could have a utility that knows how to check for a required set of keys in a map rather than this ugly if statement and use in both places.